docs/operations/node_assignment.md

```diff
@@ -140,3 +140,24 @@ An example for `tolerations` may look like this:
         operator: "Equal"
         value: "value"
         effect: "NoSchedule"
+
+## Node balancing with Descheduler
+
+In some cases we might need to rebalance the cluster on current scheduling policy 
+and load conditions. [Descheduler](https://github.com/kubernetes-sigs/descheduler) 
+can find pods, which violates e.g. scheduling decisions and evict them based on descheduler 
+policies. Kubevirt VMs are handled as pods with local storage, so by default, 
+descheduler will not evict them. But it can be easily overridden by adding special 
+annotation to the VMI template in the VM:
+
+    ```console
+    spec:
+      template:
+        metadata:
+          annotations:
+            descheduler.alpha.kubernetes.io/evict: true
```

Comment:
@davidvossel what do you  think, if there is no `emptyDir` added as a "disk" in virt-controller, we may actually deploy this annotation by default in our code base.

Reply:
@rmohr i think the annotation should be automated... but i think the situation it should be automatically added is when evictionStrategy: LiveMigrate is set and the VMI is indeed live migratable.

Reply:
I am not sure. I was expecting more something like a common baseline with pods. If no emptyDir as workload disk is present, allow the eviction.

Reply:
i thought we could migrate emptyDir contents. maybe i'm wrong though?

Reply:
Yes, if the VM is migrateable. If it is migrateable, I think we can always add it. I just meant, that I am not sure if we should exclude non-migrateable workloads by default from being recognized by the descheduler.